Makefile

```diff
@@ -60,6 +60,9 @@ help:  ## Describe useful make targets
 
 .PHONY: generate
 generate: generate-license  ## Regenerate code and license headers
+	buf generate --template buf.gen.yaml buf.build/bufbuild/protovalidate
+	buf generate --template conformance/buf.gen.yaml -o conformance/ buf.build/bufbuild/protovalidate-testing
+	./gradlew spotlessApply
```

Comment:
We shouldn't run spotless on generated code - better to just exclude those generated files from spotless (this will also make the spotless checks run a lot faster).